### ISSUE TITLE
Full image flairs support

### ISSUE THREAD

**Issue by miladmaz12** [ccrama/Slide#2322]
Examples: Look at the image flairs on r/soccer, r/hiphopheads.

Slide version: 5.5.3
Android version: 6.0.1

**Comment by ccrama** [comment]
Unfortunately not possible, it would require parsing the subreddit CSS and would not be reliable

**Comment by Scrumplex** [comment]
Is this possible by now? There is [GET /api/v1/subreddit/emojis/all](https://www.reddit.com/dev/api#GET_api_v1_{subreddit}_emojis_all) in the api. It seems like it could be used to get all emojis and then parse the flairs, so they display them.

**Comment by Alexendoo** [comment]
Yeah it's now possible assuming subreddits are using that to display flair images. This is probably the case by now
